mathics/builtin/files_io/importexport.py

```diff
@@ -1366,10 +1368,10 @@ def apply_element(self, filename, element, evaluation, options={}):
         )
 
     def apply_elements(self, filename, elements, evaluation, options={}):
-        "Import[filename_, elements_List?(AllTrue[#, NotOptionQ]&), OptionsPattern[]]"
+        "Import[filename_String, elements_List?(AllTrue[#, NotOptionQ]&), OptionsPattern[]]"
         # Check filename
-        path = filename.to_python()
-        if not (isinstance(path, str) and path[0] == path[-1] == '"'):
+        path = filename.value
+        if not isinstance(path, str):
```

Comment:
I don't think this can happen anymore

Reply:
You are correct that something isn't needed. But it is `filename.value`.  We should be checking to see that `filename` is a `String`. 

And I have changed the docstring signature to remove `_String` as well. The purpose of the `if not isinstance(path, str):` is to give an error message if the first argument is not a string. By removing that, we would remove the check altogether and silently return. That apparently was not desired. 

Also this method can be called from another method, and when that happens it defeats the signature check. 

Down the line, I have in mind revamping how all of this should work to again make it more scalable and performant. 
But this is another big change.